Superposition/Superposition.ipynb

```diff
@@ -448,7 +448,18 @@
     "\n",
     "> For example, for bit strings `[false, true, false]` and `[false, false, true]` the state required is $\\frac{1}{\\sqrt{2}}\\big(|010\\rangle + |001\\rangle\\big)$.\n",
     "\n",
-    "> If you need to define any helper functions, you'll need to create an extra code cell for it and execute it before returning to this cell."
+    "> If you need to define any helper functions, you'll need to create an extra code cell for it and execute it before returning to this cell.\n",
+    "\n",
+    "<br/>\n",
+    "<details>\n",
+    "  <summary><b>Need a hint? Click here</b></summary>\n",
+    "    \n",
+    "  Use `ControlledOnInt` and `ControlledOnBitString` library functions to apply a controlled gate on a target qubit. \n",
```

Comment:
It would be a nice idea to include the link to Q# documentation here.
```suggestion
    "  Use `[ControlledOnInt](https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.canon.controlledonint)` and `[ControlledOnBitString](https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.canon.controlledonbitstring)` library functions to apply a controlled gate on a target qubit. library functions to apply a controlled gate on a target qubit. \n",
```
```

Reply:
Nice suggestion. There are links provided later, but some people may want to go straight into the documentation.